config/main.py

```diff
@@ -1119,12 +1145,26 @@ def list_checkpoints(ctx, verbose):
 @click.option('-l', '--load-sysinfo', is_flag=True, help='load system default information (mac, portmap etc) first.')
 @click.option('-n', '--no_service_restart', default=False, is_flag=True, help='Do not restart docker services')
 @click.option('-d', '--disable_arp_cache', default=False, is_flag=True, help='Do not cache ARP table before reloading (applies to dual ToR systems only)')
+@click.option('-f', '--force', default=False, is_flag=True, help='Force config reload without system checks')
 @click.argument('filename', required=False)
 @clicommon.pass_db
-def reload(db, filename, yes, load_sysinfo, no_service_restart, disable_arp_cache):
+def reload(db, filename, yes, load_sysinfo, no_service_restart, disable_arp_cache, force):
     """Clear current configuration and import a previous saved config DB dump file.
        <filename> : Names of configuration file(s) to load, separated by comma with no spaces in between
     """
+    if not force and not no_service_restart:
+        if not _system_running():
+            click.echo("System is not up")
+            return
+
+        if not _delay_timers_elapsed():
+            click.echo("Services are not up")
+            return
+
+        if not _swss_ready():
+            click.echo("SWSS is not ready")
```

Comment:
Suggest to have the following notice to the user instead
"Orchagent container is not up. Retry later or use -f to avoid system checks"

maybe use the exact container name. SWSS is not the name the user knows.

Reply:
Done